rest-service/manager_rest/storage/resource_models.py

```diff
@@ -303,6 +303,19 @@ def agent(cls):
         return db.relationship('Agent', lazy='joined')
 
 
+class SecretsProvider(CreatedAtMixin, SQLResourceBase):
+    __tablename__ = 'secrets_providers'
+
+    name = db.Column(db.Text, nullable=False)
+    type = db.Column(db.Text, nullable=False)
+    connection_parameters = db.Column(JSONString, nullable=True)
+    updated_at = db.Column(UTCDateTime)
+
+    @hybrid_property
+    def key(self):
+        return self.name
```

Comment:
ah, i was wrong, sorry - this is entirely unnecessary, because secrets providers have both `id` and `name` already (we set both to the same value) (maybe we'll differentiate them in the future)

but anyway, `key` isn't related here